### PR TITLE
feat: use new fal.run URLs for apps

### PR DESCRIPTION
**Pull request by chamini2**

### CODE REVIEW

File: projects/fal/src/fal/apps.py [hunk 1]
```diff
@@ -11,8 +11,16 @@
...
-_URL_FORMAT = f"https://{{app_id}}.{flags.GATEWAY_HOST}/fal/queue"
-_REALTIME_URL_FORMAT = f"wss://{{app_id}}.{flags.GATEWAY_HOST}"
+_QUEUE_URL_FORMAT = f"https://queue.{flags.FAL_RUN_HOST}/{{app_id}}"
+_REALTIME_URL_FORMAT = f"wss://{flags.FAL_RUN_HOST}/{{app_id}}"
+
+
+def url_format(app_id: str, format: str) -> str:
+    if "/" not in app_id:
+        # Convert the app_id to the format used in the URL.
+        app_id = app_id.replace("-", "/", 1)
```
Comment (chamini2) [inline projects/fal/src/fal/apps.py:21]: this is for backwards compatibility

**Review by turbo1912 — DISMISSED** [review]

File: projects/fal/src/fal/apps.py [hunk 2]
```diff
@@ -186,7 +193,7 @@ def _connect(app_id: str, *, path: str = "/realtime") -> Iterator[_RealtimeConne
 
     from websockets.sync import client
 
-    url = _REALTIME_URL_FORMAT.format(app_id=app_id) + path
+    url = url_format(app_id, _REALTIME_URL_FORMAT) + path
```
Comment (isidentical) [inline projects/fal/src/fal/apps.py:196]: why not rewrite the app_id here directly so all the remaining code can stay the same?

Comment (chamini2) [reply projects/fal/src/fal/apps.py:196]: I don't want to inline

```py
    if "/" not in app_id:
        # Convert the app_id to the format used in the URL.
        app_id = app_id.replace("-", "/", 1)
```

on every format call I make

Comment (isidentical) [reply projects/fal/src/fal/apps.py:196]: no, i mean just change `app_id` here. then pass that down. we don't need to rewrite app_id everywehre

Comment (chamini2) [reply projects/fal/src/fal/apps.py]: not really sure, check the latest change?

**Comment by turbo1912** [comment]
6 different people ran into this problem in the past week :/

**Review by isidentical — APPROVED** [review]

**Pull request merged**